### ISSUE TITLE
typical time lag for updating package versions?

### ISSUE THREAD

**Issue by bbolker** [eddelbuettel/r2u#49]
I **definitely** don't want to look a gift horse in the mouth, but ... [some people are reporting problems](https://github.com/lme4/lme4/issues/756) because the current `r2u` versions of `RcppEigen` and `lme4` are out of sync (the recently superseded version of `lme4`, 1.1-34, won't work with the new version of `RcppEigen`).  I've tried to suggest workarounds, but having `lme4` updated to at least 1.1-35 would forestall a lot of queries.

Is there a general rule for how often packages get updated/time lag between CRAN release and `r2u` availability, or is it "when Dirk gets a chance"?

**Comment by eddelbuettel** [comment]
Yes the rule is ASAP but recall that I take p3m.dev / PPM as input. They skipped yesterday ("it happens") and they skip weekends by design.  It bites us at work at times too.

The alternative is to either go to r-universe (which has jammy binaries) or to compile from source.  I think on balance I am still quicker that the updates for a certain OS from Cupertino are at times at the CRAN master site.  (Guessing here, their time also seems to "vary".)

In short it is what it is.  I updated every single time there were upstream updates so I am not adding delays.

**Comment by eddelbuettel** [comment]
Also I didn't read all of your https://github.com/lme4/lme4/issues/756 but some of those installations are using r-lib and default actions and not r2u ... but are of course ham-strung by the same issue that p3m aka PPM aka RSPM doesn't "instantly" rebuilt either.    

You live by the sword of new code, you can get hurt by the sword.  At work we got a lightning bolt of a  BioC 3.18 / R 4.3.2 related breakage on data.frame conversion on the weekend.  That's the way it can go.

Not sure what else we can do here really.

**Comment by bbolker** [comment]
Thanks!

**Comment by eddelbuettel** [comment]
I'd be all ears (even quick huddle or chat) if you have thoughts about how to streamline this.  I see no low-hanging fruit but I could also be blinded by something or other...
